### ISSUE TITLE
Creating virtualenv fails on Windows due to permission error when using Python3.6

### ISSUE THREAD

**Issue by ashutoshkumr** [pypa/virtualenv#1611]
Works with version 16
----------------------
```cmd
F:\ashukuma>"C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe" -m virtualenv --version
16.0.0

F:\ashukuma>"C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe" -m virtualenv env1
Using base prefix 'C:\\Program Files (x86)\\Ixia\\Python\\3.6.4'
New python executable in F:\ashukuma\env1\Scripts\python.exe
Installing setuptools, pip, wheel...done.
```

Fails with version 20
---------------------

```cmd
F:\ashukuma>"C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe" -m pip install --upgrade virtualenv
Collecting virtualenv
  Downloading virtualenv-20.0.3-py2.py3-none-any.whl (4.6 MB)
     |████████████████████████████████| 4.6 MB 1.7 MB/s
Requirement already satisfied, skipping upgrade: appdirs<2,>=1.4.3 in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtualenv) (1.4.3)
Requirement already satisfied, skipping upgrade: six<2,>=1.12.0 in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtualenv) (1.14.0)
Requirement already satisfied, skipping upgrade: importlib-metadata<2,>=0.12; python_version < "3.8" in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtual
env) (1.5.0)
Requirement already satisfied, skipping upgrade: filelock<4,>=3.0.0 in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtualenv) (3.0.12)
Requirement already satisfied, skipping upgrade: importlib-resources<2,>=1.0; python_version < "3.7" in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtual
env) (1.0.2)
Requirement already satisfied, skipping upgrade: distlib<1,>=0.3.0 in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from virtualenv) (0.3.0)
Requirement already satisfied, skipping upgrade: zipp>=0.5 in c:\program files (x86)\ixia\python\3.6.4\lib\site-packages (from importlib-metadata<2,>=0.12; python_version < "3.
8"->virtualenv) (2.2.0)
Installing collected packages: virtualenv
  Attempting uninstall: virtualenv
    Found existing installation: virtualenv 16.0.0
    Uninstalling virtualenv-16.0.0:
      Successfully uninstalled virtualenv-16.0.0
  WARNING: The script virtualenv.exe is installed in 'C:\Program Files (x86)\Ixia\Python\3.6.4\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed virtualenv-20.0.3

F:\ashukuma>"C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe" -m virtualenv -vvv --with-traceback env1
147 setup logging to NOTSET [DEBUG report:43]
163 find interpreter for spec PythonSpec(path=C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe) [INFO builtin:44]
164 proposed PythonInfo(spec=CPython3.6.4.final.0-32, exe=C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe, platform=win32, version='3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:0
4:45) [MSC v.1900 32 bit (Intel)]', encoding_fs_io=utf-8-utf-8) [INFO builtin:50]
165 accepted PythonInfo(spec=CPython3.6.4.final.0-32, exe=C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe, platform=win32, version='3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:0
4:45) [MSC v.1900 32 bit (Intel)]', encoding_fs_io=utf-8-utf-8) [DEBUG builtin:52]
169 symlink on filesystem does work [DEBUG info:47]
170 filesystem is not case-sensitive [DEBUG info:28]
240 create virtual environment via CPython3Windows(dest=F:\ashukuma\env1, clear=False, global=False) [INFO session:51]
241 write F:\ashukuma\env1\pyvenv.cfg [DEBUG pyenv_cfg:34]
242     home = C:\Program Files (x86)\Ixia\Python\3.6.4 [DEBUG pyenv_cfg:38]
243     implementation = CPython [DEBUG pyenv_cfg:38]
243     version_info = 3.6.4.final.0 [DEBUG pyenv_cfg:38]
244     virtualenv = 20.0.3 [DEBUG pyenv_cfg:38]
245     include-system-site-packages = false [DEBUG pyenv_cfg:38]
246     base-prefix = C:\Program Files (x86)\Ixia\Python\3.6.4 [DEBUG pyenv_cfg:38]
246     base-exec-prefix = C:\Program Files (x86)\Ixia\Python\3.6.4 [DEBUG pyenv_cfg:38]
247     base-executable = C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe [DEBUG pyenv_cfg:38]
249 remove file F:\ashukuma\env1\Scripts\python.exe [DEBUG _sync:35]
251 symlink C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe to F:\ashukuma\env1\Scripts\python.exe [DEBUG _sync:41]
252 remove file F:\ashukuma\env1\Scripts\pythonw.exe [DEBUG _sync:35]
253 symlink C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe to F:\ashukuma\env1\Scripts\pythonw.exe [DEBUG _sync:41]
255 remove file F:\ashukuma\env1\Scripts\python.exe [DEBUG _sync:35]
Traceback (most recent call last):
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\__main__.py", line 47, in <module>
    run_with_catch()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\__main__.py", line 36, in run_with_catch
    run(args, options)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\__main__.py", line 19, in run
    session = cli_run(args, options)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\run\__init__.py", line 23, in cli_run
    session.run()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\session.py", line 45, in run
    self._create()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\session.py", line 52, in _create
    self.creator.run()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\create\creator.py", line 160, in run
    self.create()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\create\via_global_ref\builtin\via_global_self_do.py", line 78, in create
    src.run(self, self.symlinks)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\create\via_global_ref\builtin\ref.py", line 115, in run
    method(self.src, dest)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\util\path\_sync.py", line 40, in symlink
    ensure_safe_to_do(src, dest)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\site-packages\virtualenv\util\path\_sync.py", line 36, in ensure_safe_to_do
    dest.unlink()
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\pathlib.py", line 1262, in unlink
    self._accessor.unlink(self)
  File "C:\Program Files (x86)\Ixia\Python\3.6.4\lib\pathlib.py", line 387, in wrapped
    return strfunc(str(pathobj), *args)
PermissionError: [WinError 5] Access is denied: 'F:\\ashukuma\\env1\\Scripts\\python.exe'
```

Pip List
--------
```cmd
F:\ashukuma>"C:\Program Files (x86)\Ixia\Python\3.6.4\python.exe" -m pip list
Package             Version
------------------- ---------
appdirs             1.4.3
asn1crypto          0.24.0
certifi             2018.4.16
cffi                1.11.5
chardet             3.0.4
cryptography        2.2.2
distlib             0.3.0
filelock            3.0.12
idna                2.6
importlib-metadata  1.5.0
importlib-resources 1.0.2
ixnetwork-restpy    1.0.43
pip                 20.0.2
pycparser           2.18
pyOpenSSL           17.5.0
requests            2.18.4
setuptools          28.8.0
six                 1.14.0
urllib3             1.22
virtualenv          20.0.3
websocket-client    0.47.0
zipp                2.2.0
```

**Comment by gaborbernat** [comment]
Can you please attach a list of files present within C:\Program Files (x86)\Ixia\Python\3.6.4? Thanks!

**Comment by ashutoshkumr** [comment]
Attached recursive list of files inside  C:\Program Files (x86)\Ixia\Python\3.6.4 (Check out.log).
[out.log](https://github.com/pypa/virtualenv/files/4203694/out.log)

**Comment by gaborbernat** [comment]
@ashutoshkumr can you test if this is still the case with https://pypi.org/project/virtualenv/20.0.9/?

**Comment by ashutoshkumr** [comment]
Hello, thank you. This is fixed in 20.0.21.
